<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryDialog</class>
 <widget class="QDialog" name="QueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Things</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListWidget" name="m_listWidget">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,0.5)</string>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragOnly</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::IgnoreAction</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="m_addButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/img/setting.png</normaloff>:/img/setting.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_editButton">
            <property name="text">
             <string>Edit</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/img/color.png</normaloff>:/img/color.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_exceptButton">
            <property name="text">
             <string>Except</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/img/exception.png</normaloff>:/img/exception.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="m_deleteButton">
            <property name="text">
             <string>Delete</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_deleteAllButton">
            <property name="text">
             <string>Delete All</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/img/deleteall.png</normaloff>:/img/deleteall.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Files</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="DragableListWidget" name="m_fileListWidget">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,0.5)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="m_pickColorButton">
        <property name="text">
         <string>Color</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/img/setting.png</normaloff>:/img/setting.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_colorLabel">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_okButton">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/img/ok.png</normaloff>:/img/ok.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragableListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">dragablelistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_okButton</sender>
   <signal>clicked()</signal>
   <receiver>QueryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
